lib/main.js

```diff
@@ -608,17 +587,16 @@ function GetSVG(result) {
   //
   //  Add the speech text and mark the SVG appropriately
   //
-  if (data.speakText) {
-    ID++; var id = "MathJax-SVG-"+ID;
-    svg.removeAttribute("aria-label");
-    svg.setAttribute("aria-labelledby",id+"-Title "+id+"-Desc");
-    for (var i=0, m=svg.childNodes.length; i < m; i++)
-      svg.childNodes[i].setAttribute("aria-hidden",true);
-    var node = MathJax.HTML.Element("desc",{id:id+"-Desc"},[result.speakText]);
-    svg.insertBefore(node,svg.firstChild);
-    node = MathJax.HTML.Element("title",{id:id+"-Title"},["Equation"]);
-    svg.insertBefore(node,svg.firstChild);
-  }
+  for (var i=0, m=svg.childNodes.length; i < m; i++)
+    svg.childNodes[i].setAttribute("aria-hidden",true);
+  // Note: if aria-label exists, getSpeech preserved it in speakText
+  // remove aria-label since labelled-by desc is preferred
+  svg.removeAttribute("aria-label");
+  ID++; var id = "MathJax-SVG-"+ID;
```

Comment:
No reason. Just copy and paste from the original (where a title was also present and thus you had split this up).